flow/designs/sky130hd/chameleon_hier/DMC_32x16HC/config.mk

```diff
@@ -0,0 +1,44 @@
+
+undefine BLOCKS
+undefine ADDITIONAL_LEFS
+undefine ADDITIONAL_GDS 
+undefine DONT_USE_SC_LIB
+undefine RCX_RULES
+
+export TOP_NICKNAME = chameleon_hier
+export TOP_DIR = ./designs/$(PLATFORM)/${TOP_NICKNAME}
+
+export DESIGN_NAME = DMC_32x16HC
+export DESIGN_NICKNAME = ${TOP_NICKNAME}_${DESIGN_NAME}
+export PLATFORM    = sky130hd
+export RTL_DIR  = ./designs/src/${TOP_NICKNAME}/rtl
+
+export VERILOG_FILES = \
+                         ${RTL_DIR}/IPs/DFFRAMBB.v \
+	                 ${RTL_DIR}/IPs/DMC_32x16HC.v
+
+			 
+export SDC_FILE          = ${TOP_DIR}/${DESIGN_NAME}/constraint.sdc
+#export RCX_RULES     = $(PLATFORM_DIR)/rcx_patterns.rules
+
+export PDN_CFG = ${TOP_DIR}/${DESIGN_NAME}/pdn.cfg
+
+export ABC_CLOCK_PERIOD_IN_PS = 10000
+export ABC_DRIVER_CELL = sky130_fd_sc_hd__buf_1
+export ABC_LOAD_IN_FF = 3
+
+
+# Point to the RC file
+export SETRC_FILE = $(PLATFORM_DIR)/setRC.tcl
```

Comment:
Is this used anywhere? Nothing appears to reference that variable

Reply:
I've removed it